view-user/src/pages/locales/ja.ts

```diff
@@ -0,0 +1,9 @@
+export const ja = {
+  PRIZE_BUTTON: "景品",
+  NUMBER_BUTTON: "番号一覧",
+  SUB_TITLE_NUMBER: "BINGO Number",
```

Comment:
```suggestion
  SUB_TITLE_NUMBER: "抽選された番号",
```
ここだけ翻訳されない方が不自然だし日本語にしちゃってもいいかも

Reply:
修正しました。